model/lib/extensions/presentation_email_extension.dart

```diff
@@ -243,4 +244,19 @@ extension PresentationEmailExtension on PresentationEmail {
       mailboxContain: mailboxContain
     );
   }
+
+  bool isBelongToOneOfTheMailboxes(List<MailboxId> mailboxIdsSource) {
+    final mapMailboxIds = mailboxIds;
+    mapMailboxIds?.removeWhere((key, value) => !value);
+
+    if (mapMailboxIds?.isNotEmpty == true) {
+      final listMailboxId = mapMailboxIds!.keys.toList();
+      log('PresentationEmailExtension::isBelongToOneOfTheMailboxes():listMailboxId: $listMailboxId');
+      final listMailboxIdValid = listMailboxId.where((mailboxId) => mailboxIdsSource.contains(mailboxId));
+      log('PresentationEmailExtension::isBelongToOneOfTheMailboxes():listMailboxIdValid: $listMailboxIdValid');
+      return listMailboxIdValid.isNotEmpty;
```

Comment:
How about?
```suggestion
      return mapMailboxIds!.keys.where((mailboxId) => mailboxIdsSource.contains(mailboxId)).isNotEmpty
```

Reply:
> How about?

I wanted it to make more sense to the reader, so I wrote it like that. Do you find it confusing? @hoangdat